apps/ideas/forms.py

```diff
@@ -270,7 +270,7 @@ class NetworkAndCommunitySectionForm(CoWorkersEmailsFormMixin, BaseForm):
         help_text=COWORKERS_HELP,
         label=COWORKERS_TITLE)
     confirm_publicity = forms.BooleanField(label=CONFIRM_PUBLICITY_LABEL)
-    accept_conditions = forms.BooleanField(label='')
+    accept_conditions = forms.BooleanField(label=ACCEPT_CONDITIONS_LABEL)
```

Comment:
This was empty on purpose, because the label is filled in the __init__ a few lines below. It depends on the wagtail settings to add the terms_of_use_page. And its added with this weird helper-function. All a bit complicated. :roll_eyes: